xbmc/storage/MediaManager.cpp

```diff
@@ -716,29 +716,26 @@ CMediaManager::DiscInfo CMediaManager::GetDiscInfo(const std::string& mediaPath)
   if (mediaPath.empty())
     return info;
 
-  // Try finding VIDEO_TS/VIDEO_TS.IFO - this indicates a DVD disc is inserted
+  // Check for DVD discs
   std::string pathVideoTS = URIUtils::AddFileToFolder(mediaPath, "VIDEO_TS");
```

Comment:
This line isn't needed anymore as far as I can tell

Reply:
fixed